src/shared/components/palette/complex/read-complex-field-collection-table.component.ts

```diff
@@ -154,15 +120,51 @@ export class ReadComplexFieldCollectionTableComponent extends AbstractFieldReadC
     }
   }
 
-  private isSortAscending(column: any): boolean {
-    return !(column.sortOrder === SortOrder.UNSORTED || column.sortOrder === SortOrder.DESCENDING);
+  public sortWidget(column: any): string {
+    return ReadComplexFieldCollectionTableComponent.isSortAscending(column) ? '&#9660;' : '&#9650;';
+  }
+
+  private populateHorizontalLabels(labelsHorizontal: { [p: string]: any },
+                                   allLabels: { [p: string]: any },
+                                   labelsVertical: { [p: string]: any }) {
+    for (let id of this.columns) {
+      labelsHorizontal[id.trim()] = allLabels[id.trim()];
```

Comment:
_Nitpick:_
Why do `id.trim()` on every single instance it's being used rather than `const trimId: string = id.trim();` in the line above?